src/lwc/geGiftEntryFormApp/geGiftEntryFormApp.js

```diff
@@ -197,6 +219,11 @@ export default class GeGiftEntryFormApp extends NavigationMixin(LightningElement
         return format(geBatchGiftsHeader, [this.batchName]);
     }
 
+    get shouldDisplayExpiredAuthorizationWarning() {
+        return this.batchTotals.hasPaymentsWithExpiredAuthorizations 
+            && !this._hasDisplayedExpiredAuthorizationWarning;
+    }
```

Comment:
No need for a getter here I think. Just a normal function.
```suggestion
    shouldDisplayExpiredAuthorizationWarning() {
        return this.batchTotals.hasPaymentsWithExpiredAuthorizations 
            && !this._hasDisplayedExpiredAuthorizationWarning;
    }
```